spark/src/main/scala/io/delta/sharing/spark/DeltaSharingSource.scala

```diff
@@ -244,6 +245,7 @@ case class DeltaSharingSource(
           f.id -> f.url
         }.toMap
```

Comment:
As discussed, we should update the sortedFetchedFiles as well in order to avoid redundant refreshes.

Reply:
updated.